OWNERS_ALIASES

```diff
@@ -7,7 +7,6 @@ aliases:
   sdk-admins:
   - estroz
   - joelanford
-  - jmccormick2001
```

Comment:
I removed him from the .cncf-maintainers erroneously. That is a generated file based on this